### ISSUE TITLE
question: How I create a secret key from a string?

### ISSUE THREAD

**Issue by Nisgrak** [panva/paseto#2]
Hi!

I try to encrypt a payload with V2, I want to encrypt with key 'example', how can make it work?

Always I give the error
```
TypeError: v2.local secret key must be 256 bytes long symmetric key
```

Thanks and nice work 😄

**Comment by panva** [comment]
Hi @Nisgrak, you need a secret that's 256 bytes :) `"example"` isn't

```js
Buffer.from('example').byteLength
// 7
```

**Comment by Nisgrak** [comment]
Okey, I add chars to the key then I have 256 bytes, but the error is the same.

```js
Buffer.from('exampleeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee').byteLength;
// 256
```

**Comment by panva** [comment]
🤦‍♂ sorry, 32 bytes, 256 bits. I fixed the error message in 1.0.2
